### PR TITLE
Fix: Added `self` as the first parameter at places where it is missing

### PR DESCRIPTION
**Pull request by Sai-Suraj-27**
# PR Description
Added `self` as the first parameter at 2 places where it is missing.

## Checklist
- [ ] Did you add a function?
- [ ] Did you add the tests?
- [ ] Did you follow the steps we provided?

### Socials:
https://twitter.com/Sai_Suraj_27

### CODE REVIEW

File: ivy/__init__.py [hunk 1]
```diff
@@ -435,11 +435,11 @@ def assert_has_rank(self, rank):
         if self.rank not in (None, rank):
             raise ValueError("Shape %s must have rank %d" % (self, rank))
 
-    def unknown_shape(rank=None, **kwargs):
```
Comment (KareemMAX) [inline ivy/__init__.py:438]: I guess it is reasonable to have this function as static. As it produce `unknown_shape` from `rank`

File: ivy/__init__.py [hunk 2]
```diff
@@ -463,7 +463,7 @@ def with_rank_at_most(self, rank):
         else:
             return self
 
-    def as_shape(shape):
```
Comment (KareemMAX) [inline ivy/__init__.py:466]: I guess it is reasonable to have this function as static.

**Review by KareemMAX — CHANGES_REQUESTED** [review]
Hey @Sai-Suraj-27,
Thanks for the contribution, just made a few comments.

File: ivy/__init__.py (hunk 2 above)
Comment (Sai-Suraj-27) [reply ivy/__init__.py:466]: Ok, sir. I will retrieve these 2 changes. 👍

**Review by KareemMAX — APPROVED** [review]

**Pull request merged**